File: 01-Assignment-02-02-2026/Using_C/program_35.c
/* Program 35: Find the sum of even numbers */

#include <stdio.h>

int main() {
    int n, sum = 0;
    printf("Enter a number: ");
    scanf("%d", &n);

    if (n < 2) {
        printf("Sum -> 0");
        return 0;
    }

    for (int i = 2; i <= n; i += 2) {
        sum += i;
    }

    printf("Sum -> %d", sum);
    return 0;
}

File: 01-Assignment-02-02-2026/Using_Java/Program35.java
/*
Program 35
Java program to find the sum of even numbers
*/

import java.util.Scanner;

public class Program35 {

    public static boolean isEven(int n){
        return n%2 == 0;
    }
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter a number : ");
        int num = scanner.nextInt() , sum = 0;

        for(int i = 0;i <= num;i++){
            if (isEven(i))
                sum+=i;

        }
        System.out.println("Sum of all even numbers upto " + num + " is " + sum);
        scanner.close();
    }

}
